### ISSUE TITLE
3D displacement image

### ISSUE THREAD

**Issue by yutianp** [yutianp/akiraexperiment#1]
Currently working on a 3D image effect using pixi.js as a workaround for creating a "3D" scene in HTML. Not sure how this is gonna turn out and will continue to work on the actual three.js in case this doesn't look good.

The tutorial I am currently watching: https://www.youtube.com/watch?v=FgN2EENWPFc

End result I want (minus the blurry circle thing): https://www.fotospatzek.at/Home
tutorial for this page: https://medium.com/site-effects-under-the-hood/www-fotospatzek-at-under-the-hood-9ccbda7d75d1

Currently getting this error and my images won't load.
![image](https://user-images.githubusercontent.com/70179543/95796478-79abad80-0d49-11eb-841d-2a7713d07111.png)

**Comment by sehsarah** [comment]
> End result I want (minus the blurry circle thing): https://www.fotospatzek.at/Home

Can u add a screenshot and video showing this end result you want? I don't think the site is loading properly for me

**Comment by sehsarah** [comment]
Ah i see, it's loaded for me now. So can u describe further what you're trying to do. Just render a scene and be able to pan around it?

**Comment by sehsarah** [comment]
Add this stuff to your root repo folder, not within the named folder https://github.com/yutianp/akiraexperiment/tree/main/pixi%20min%20js

**Comment by sehsarah** [comment]
Those links and your code example are using images, I thought you had rendered a 3D blender model and you wanted to render that in the browser? That's what's done in the experiment repo I added you to, over here https://meidestudio.github.io/3dexperiments/2/

If you export your model as a `.dae` file you should be able to upload it within that example and play around. Is that becoming more similar to what you're after?

**Comment by yutianp** [comment]
Yup I tried your 3d experiment thing using the .dae file. It's cool to see my 3d models in a browser however I don't think I can add textures and materials to the model as well as control the lighting. I'm currently also working on a three.js exporting my model as a .glTF which includes textures and materials. 

I'm creating the 3D image thing with pixi just in case I run out of time as the three.js one is gonna take a hefty amount of time.

**Comment by yutianp** [comment]
> Ah i see, it's loaded for me now. So can u describe further what you're trying to do. Just render a scene and be able to pan around it?

But yeah I just wanna be able to slightly orbit around the models which can be achieved with the displacement jpg technique.

This is ideally what I'm trying to achieve with three.js:
https://garden-eight.com/

**Comment by sehsarah** [comment]
> however I don't think I can add textures and materials to the model as well as control the lighting. I'm currently also working on a three.js exporting my model as a .glTF which includes textures and materials.

Cool this sounds good. I reckon focus on utilising the models in browser, even if you can't get textures and lighting perfect 

Will the displacement jpg idea use your models though or jpgs?

**Comment by yutianp** [comment]
the displacement jpg idea uses rendered jpgs of my models. I create the displacement maps by rendering out a separate jpg using the alpha channels on the same 3d model so that it is an accurate displacement map
